superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx

```diff
@@ -12,10 +12,10 @@ import { getExploreUrl } from '../../explore/exploreUtils';
 import * as actions from '../actions';
 
 const CHART_TYPES = [
```

Comment:
i think we should move this to the `visTypes` constant in the store so we are not duplicating the labels, and can make use of these additional flags elsewhere as well.
https://github.com/airbnb/superset/blob/master/superset/assets/javascripts/explore/stores/visTypes.js#L75